peps/pep-0734.rst

```diff
@@ -0,0 +1,399 @@
+PEP: 734
+Title: Multiple Interpreters in the Stdlib
+Author: Eric Snow <ericsnowcurrently@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 06-Nov-2023
+Python-Version: 3.13
+Replaces: 554
+
+.. note::
+   This PEP is essentially a continuation of :pep:`554`.  That document
+   had grown a lot of ancillary information across 7 years of discussion.
+   This PEP is a reduction back to the essential information.  Much of
+   that extra information is still valid and useful, just not in the
+   immediate context of the specific proposal here.
+
+Abstract
+========
+
+I propose that we add a new module, "interpreters", to the standard
+library, to make the existing multiple-interpreters feature of CPython
+more easily accessible to Python code.  This is particularly relevant
+now that we have a per-interpreter GIL (:pep:`684`) and people are
+more interested in using multiple interpreters.  Without a stdlib
+module, users are limited to the `C-API`_, which restricts how much
+they can try out and take advantage of multiple interpreters.
+
+.. _C-API:
+   https://docs.python.org/3/c-api/init.html#sub-interpreter-support
+
+
+Rationale
+=========
+
+The ``interpreters`` module will provide a high-level interface to the
+multiple interpreter functionality.  Since we have no experience with
+how users will make use of multiple interpreters in Python code, we are
+purposefully keeping the initial API as lean and minimal as possible.
+The objective is to provide a well-considered foundation on which we may
+add more-advanced functionality later.
```

Comment:
```suggestion
add more advanced functionality later.
```

Reply:
The hyphen was intentional, but your comment demonstrates that it may be awkward. 😄  I'll reword the phrase.